bson/src/test/unit/org/bson/json/JsonReaderTest.java

```diff
@@ -119,6 +132,16 @@ public void testDateTimeShell() {
     }
 
     @Test
+    public void testDateTimeShellNow() {
+        long currentTime = new Date().getTime();
+        String json = "ISODate()";
```

Comment:
That's because you're calling "new Date()" instead of "Date()"

JavaScript is very weird sometimes.